app/src/features/projects/view/components/TeamMember.tsx

```diff
@@ -0,0 +1,97 @@
+import { mdiCrown, mdiPencilOutline } from '@mdi/js';
+import Icon from '@mdi/react';
+import { Box, Stack, Typography } from '@mui/material';
+import assert from 'assert';
+import { ProjectContext } from 'contexts/projectContext';
+import { useContext } from 'react';
+
+export interface IProjectParticipantsRoles {
+  display_name: string;
+  roles: string[];
+  initials: string;
+}
+
+const TeamMembers = () => {
+  const projectContext = useContext(ProjectContext);
+
+  // Project data must be loaded by a parent before this component is rendered
+  assert(projectContext.projectDataLoader.data);
+
+  // Define a custom sorting order for roles
+  const roleOrder: { [key: string]: number } = {
+    Coordinator: 1,
+    Collaborator: 2,
+    Observer: 3
+  };
+
+  const projectTeamMembers: IProjectParticipantsRoles[] = projectContext.projectDataLoader.data.projectData.participants
```

Comment:
This should be memoized. To that end, we may as well also calculate the user's avatar color (see my other comment) in here as well.
```javascript
const projectTeamMembers: (IProjectParticipantsRole & { avatarColor: string })[] = useMemo(() => {
   // Logic to get their initials etc goes here
}, [projectContext.projectDataLoader.data.projectData.participants]);
```